### DISCUSSION TITLE
Prevent scroll when updating search params (eg. for infinite scroll page param)

### DISCUSSION THREAD

**Discussion by apalumbo** [remix-run/remix#4992] [Show and tell / tips]
The following code is not working on remix , it seems that preventScrollReset is not jet handled (looking at the code the navigator used does not accept the options parameter for push and pop like the one created in RouterProvider in react-router)

```
setSearchParams({page: `10`}, { replace: true, preventScrollReset: true });
```

This version works i found this work around in another discussion and I thought it would be useful to share
```
setSearchParams({page: `10`}, { replace: true, state: { scroll: false } });
```

**Comment by pedrorolando** [comment]
Did you ever find a solution for this?

**Reply by apalumbo** [reply]
No, it is still an issue on the latest version?
